apps/meteor/client/sidebar/sections/OmnichannelSection.tsx

```diff
@@ -61,6 +64,7 @@ const OmnichannelSection = (props: typeof Box): ReactElement => {
 				{showOmnichannelQueueLink && <Sidebar.TopBar.Action icon='queue' title={t('Queue')} onClick={(): void => handleRoute('queue')} />}
 				{isCallEnabled && <OmnichannelCallToggle />}
 				<Sidebar.TopBar.Action {...availableIcon} onClick={handleAvailableStatusChange} />
+				{isCallEnabled && isCallReady && isEnterprise && <OmnichannelMakeCall />}
```

Comment:
Here you need to be careful because the hook `useHasLicence` returns `true | false | 'loading'`. 
Considering one of the values is a string (truthy) it might cause an issue here. One possible solution is to enforce the type: `const isEnterprise = useHasLicence('voip-enterprise') === true`

Reply:
Sure. I will fix this. Importantly, this line will be removed once the dialpad is wired. @tiagoevanp please take a note of this line.